src/main/java/edu/harvard/iq/dataverse/branding/BrandingUtil.java

```diff
@@ -1,22 +1,37 @@
 package edu.harvard.iq.dataverse.branding;
 
+import edu.harvard.iq.dataverse.DataverseServiceBean;
 import edu.harvard.iq.dataverse.util.BundleUtil;
 import java.util.Arrays;
+
+import javax.ejb.EJB;
+import javax.ejb.Singleton;
 import javax.mail.internet.InternetAddress;
 
+@Singleton
 public class BrandingUtil {
 
-    public static String getInstallationBrandName(String rootDataverseName) {
-        return rootDataverseName;
+    @EJB
+    static DataverseServiceBean dataverseService;
+    
+    public static String getInstallationBrandName() {
+        //ToDo #7387 which will make this call return something different than getRootDataverseCollectionName() 
+        return dataverseService.getRootDataverseName();
+    }
+    
+    //Convenience to access root name without injecting dataverseService (e.g. in DatasetVersion)
+    public static String getRootDataverseCollectionName() {
+        return dataverseService.getRootDataverseName();
```

Comment:
Shouldn't we do this more consistent then and go for changing the function in `DataverseServiceBean`, too? (I am aware of the consequences)